client/dev-tools-promo/index.tsx

```diff
@@ -0,0 +1,19 @@
+import page from '@automattic/calypso-router';
+import { makeLayout, render as clientRender } from 'calypso/controller';
+import { navigation, siteSelection, sites } from 'calypso/my-sites/controller';
+import { siteDashboard } from 'calypso/sites-dashboard-v2/controller';
+import { DOTCOM_OVERVIEW } from 'calypso/sites-dashboard-v2/site-preview-pane/constants';
+import { devToolsPromo } from './controller';
+
+export default function () {
+	page( '/dev-tools-promo', siteSelection, sites, makeLayout, clientRender );
+	page(
+		'/dev-tools-promo/:site',
+		siteSelection,
+		navigation,
```

Comment:
We have to redirect to the Overview tab when the selected site changes to the atomic site 🙂

Reply:
ahh I understand, thanks